solc/CommandLineInterface.cpp

```diff
@@ -610,6 +614,23 @@ Allowed options)",
 
 bool CommandLineInterface::processInput()
 {
+	if (m_args.count(g_argStandardJSON))
+	{
+		string input;
+		while (!cin.eof())
```

Comment:
There's none in `CommonIO`. If there's a sample code somewhere I'm happy to use that. This came from `CommandLineInterface::readInputFilesAndConfigureRemappings`.